src/java/picard/analysis/CollectAlignmentSummaryMetrics.java

```diff
@@ -64,26 +64,33 @@
  * @author Doug Voet (dvoet at broadinstitute dot org)
  */
 @CommandLineProgramProperties(
-        usage = CollectAlignmentSummaryMetrics.USAGE_SUMMARY + CollectAlignmentSummaryMetrics.USAGE_DETAILS,
```

Comment:
The pattern for how the documentation is specified should be the same for all tools, as follows: 

    @CommandLineProgramProperties(
        usage = tool.USAGE_SUMMARY + tool.USAGE_DETAILS,
        usageShort = tool.USAGE_SUMMARY,
        programGroup = Metrics.class
    )

    public class Tool extends TypeOfProgram {
        static final String USAGE_SUMMARY = "one-line description";
        static final String USAGE_DETAILS = "full details go here";

        // program code goes here
    }

You have it right in some of them but not all; this should be made the same everywhere.